app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,53 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    policy_contents.each do |content|
+      find_associations!(content)
+      policy_name = content[:miq_policy].try(:name)
+
+      rec = find_by_pkey(policy_name, content[:success_sequence], content[:failure_sequence])
+      if rec.nil?
+        create_seed_record(policy_name, content)
+      else
+        update_seed_record(rec, policy_name, content)
```

Comment:
I started with a method to seed one policy content but I had to split it further to find_by_pkey, create_seed_record and update_seed_record to get it simple enough for rubocop's taste